### ISSUE TITLE
Unable to access imported items dynamically

### ISSUE THREAD

**Issue by rhys-vdw** [benmosher/eslint-plugin-import#456]
Sorry, don't know the correct jargon for this issue title. Issue I'm hitting:

```js
import * as Module from './Module';

function getItem(name) {
  const item = Module[name];
}
```

error:
```console
error    Unable to validate computed reference to imported namespace 'Module'  import/namespace
```

So I don't know if this is legal by the spec. Babel doesn't object. Perhaps this kind of property access should have its own rule. If it is not permitted by the spec, then it could be an error. If it is legal, then it could be a warning by default? As it is I need to disable `import/namespace` entirely to allow this.

**Comment by benmosher** [comment]
Ah, yeah... it's valid, but the rule is basically saying "you're on your own, I can't lint computed references".

Should probably be a separate rule entirely, as some users (certainly you!) would want static reference linting without getting tripped on this.

I think I'll split this out as a separate rule in `v2`, but for now as a workaround you can `// eslint-disable-line import/namespace`? How does that sound?

**Comment by rhys-vdw** [comment]
> but for now as a workaround you can // eslint-disable-line import/namespace? How does that sound?

Perfect!

**Comment by benmosher** [comment]
Just published the `allowComputed` rule option with v1.13.0 😎
